posthog/settings/payments.py

```diff
@@ -0,0 +1,3 @@
+from posthog.settings.utils import get_from_env
+
+STRIPE_PUBLIC_KEY = get_from_env("STRIPE_PUBLIC_KEY", None)
```

Comment:
@zlwaterfield was it a purposeful choice this wasn't in settings? 

i needed to move it here to make it easy to set a value in tests

Reply:
No I was just copying existing patterns and it wasn't needed anything else in the python codebase.